pkg/ddl/internal/session/session_pool.go

```diff
@@ -72,6 +72,7 @@ func (sg *Pool) Get() (sessionctx.Context, error) {
 	}
 	ctx.GetSessionVars().SetStatusFlag(mysql.ServerStatusAutocommit, true)
 	ctx.GetSessionVars().InRestrictedSQL = true
+	ctx.GetSessionVars().StmtCtx.SetTimeZone(ctx.GetSessionVars().Location())
```

Comment:
To make sure the timezone info are the same in session and statement context. PTAL @tangenta

Reply:
lgtm